### PR TITLE
Please review, open to additions and modifications before merging to master.

### PR DESCRIPTION
**Pull request by dreinavarro**

### CODE REVIEW

File: general/source-code-conventions.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Source code conventions
+Adhere to these general source code conventions unless otherwise specified in language specific guide.
+
+## Files
+- Use language specific indentation.
+- If you have to use tabs, do not mix tabs and spaces in a single project.
```
Comment (alecguintu) [inline general/source-code-conventions.md:6]: So we'd use both tabs and spaces for convention? Just per project though?

Comment (dreinavarro) [reply general/source-code-conventions.md:6]: Per language specific guidelines. Will try to post guidelines on each language.

Comment (dreinavarro) [reply general/source-code-conventions.md:6]: @angelofrancisco pareview naman neto

File: general/source-code-conventions.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- All files end with an empty line.
```
Comment (angelofrancisco) [inline general/source-code-conventions.md:7]: All files na user created kailangan may extra space sa dulo? What's the reason for this and how do we handle IDE/system generated files that are user accessible?

Comment (dreinavarro) [reply general/source-code-conventions.md:7]: cleanliness lang parang margins

@alecguintu any comment

**Review by angelofrancisco — APPROVED** [review]

File: general/source-code-conventions.md (hunk 2 above)
Comment (dreinavarro) [reply general/source-code-conventions.md:7]: http://stackoverflow.com/a/729795 @angelofrancisco

Comment (angelofrancisco) [reply general/source-code-conventions.md:7]: Okay, noted @dreinavarro

**Pull request merged**